quickstarts/pomi-stats/dashboards/pomi-stats.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
I just noticed this is still the example dashboard 😄  Could you upload yours?

Reply:
done, not sure how I missed that 🤦‍♂️